slack-base/src/main/java/com/hubspot/slack/client/models/files/SlackFileType.java

```diff
@@ -9,6 +9,7 @@
   TEXT("text"),
   GIF("gif"),
   CSV("csv"),
+  JPG("jpg")
```

Comment:
```suggestion
  JPG("jpg"),
```
nit: comma at the end helps keep future diffs clean